scripts/reset-all.sh

```
@@ -33,6 +39,51 @@ for arg in "$@"; do
     esac
 done
 
+# Function to check if database host is local
+is_local_host() {
+    local db_url="$1"
+    # Extract host from database URL
+    # URL format: postgresql://user:pass@host:port/db
+    local host=$(echo "$db_url" | sed -n 's/.*@\([^:]*\):.*/\1/p')
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)